packages/next/src/plugins/plugin.ts

```diff
@@ -152,6 +155,18 @@ function getStartTargetConfig(options: NextPluginOptions, projectRoot: string) {
   return targetConfig;
 }
 
+function getStaticServeTargetConfig(options: NextPluginOptions) {
+  const targetConfig: TargetConfiguration = {
+    executor: '@nx/web:file-server',
+    options: {
+      buildTarget: options.buildTargetName,
```

Comment:
We should specify the port options here as well so users can inspect it via PDV, and know how to override it.

Reply:
hmmm wouldn't they just override it inside `project.json` 
This is already being shown in the PDV.

<img width="720" alt="Screenshot 2024-02-15 at 10 08 11 AM" src="https://github.com/nrwl/nx/assets/338948/9b6ff7a5-29d2-49fc-b7a9-08d0d902c93d">